src/game/game.c

```diff
@@ -187,10 +187,18 @@ void game_run(void)
     }
 }
 
-void game_draw(void)
+static int need_force_draw = 0;
+void game_draw()
 {
-    window_draw(0);
+    window_draw(need_force_draw);
```

Comment:
You don't need a `need_force_draw` or a `game_force_draw`.

`window_request_refresh()` should do the trick.